src/console-services.yml

```diff
@@ -1,31 +1,63 @@
 ---
-- command: acm
-  description: 'Certificate Manager - Provision, Manage, and Deploy SSL/TLS Certificates'
-  url: 'https://console.aws.amazon.com/acm/home'
+- command: alexa
+  description: 'Alexa for Business - Empower your organization with voice'
+  url: 'https://console.aws.amazon.com/a4b/home'
+
+- command: amplify
+  description: 'Amplify - Develop & deploy cloud-powered mobile and web apps'
+  url: 'https://console.aws.amazon.com/amplify/home'
 
 - command: apigateway
   description: 'API Gateway - Build, Deploy and Manage APIs'
   url: 'https://console.aws.amazon.com/apigateway/home'
 
 - command: appstream
-  description: 'AppStream - Low Latency Application Streaming'
-  url: 'https://console.aws.amazon.com/appstream/home'
+  description: 'AppStream 2.0 - Stream desktop applications securely to any web browser'
+  url: 'https://console.aws.amazon.com/appstream2/home'
+
+- command: appsync
+  description: 'AppSync - Real-Time Data Sync Using GraphQL for Mobile & Web Apps, Online or Offline'
+  url: 'https://console.aws.amazon.com/appsync/home'
+
+- command: artifact
+  description: 'Artifact - AWS Compliance Reports and Agreements'
+  url: 'https://console.aws.amazon.com/artifact/home'
 
 - command: athena
-  description: 'Athena - Serverless Interactive Query Service'
+  description: 'Athena - Query Data in S3 using SQL'
   url: 'https://console.aws.amazon.com/athena/home'
 
+- command: awsautoscaling
+  description: 'Auto Scaling - AWS Auto Scaling enables you to quickly scale your entire application on AWS'
+  url: 'https://console.aws.amazon.com/awsautoscaling/home'
+
+- command: backup
+  description: 'Backup - centrally manage and automate backups across AWS services'
+  url: 'https://console.aws.amazon.com/backup/home'
+
 - command: batch
-  description: 'Batch - Fully Managed Batch Processing at Any Scale'
+  description: 'Batch - Run Batch Jobs at Any Scale'
   url: 'https://console.aws.amazon.com/batch/home'
 
-- command: billing
-  description: 'Billing - Billing & Cost Management'
-  url: 'https://console.aws.amazon.com/billing/home'
```

Comment:
why was this removed? this seems to still be a valid link

Reply:
I think I renamed this to budgets because billing doesn't actually have an icon. Will add it back in using the aws icon.